### DISCUSSION TITLE
asdf dotnet-core support

### DISCUSSION THREAD

**Discussion by BottlecapDave** [renovatebot/renovate#30430] [Suggest an Idea]
### Tell us more.

It would be good if renovate could support [dotnet-core](https://github.com/emersonsoares/asdf-dotnet-core) as part of the [asdf manager](https://docs.renovatebot.com/modules/manager/asdf/). It looks like renovate already has a data source for the dotnet sdk/runner, so looks to be a simple addition (unless i've misunstood something in the implementation).

I've started looking into it, but wanted to raise this before raising a PR as per the guidelines.

**Comment by rarkins** [comment]
PRs welcome for asdf manager additions
